src/backend/Coq.mli

```diff
@@ -9,19 +9,21 @@ Copyright 2015 Guillaume Bury
     corresponding to the resolution proofs output by the
     sat solver. *)
 
-module type S = Backend_intf.S
 (** Interface for exporting proofs. *)
+module type S = Backend_intf.S
 
 module type Arg = sig
   (** Term printing for Coq *)
 
   type hyp
   type lemma
-  type assumption
+
   (** The types of hypotheses, lemmas, and assumptions *)
+  type assumption
```

Comment:
Again, ocamlformat moved some doc comment before the type def, which, particularly in this case, hurts readability.